src/services/weather.js

```diff
@@ -5,21 +5,25 @@ import _ from 'lodash';
 const weatherDataURL = 'https://api.openweathermap.org/data/2.5/weather';
 const weatherImageURL = 'https://openweathermap.org/img/w/';
 const imageExtension = '.png';
+const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+const degreeSign = '°';
 
 function convertKelvinToCelsius(temperature) {
-  return (temperature - 273.15).toFixed(1);
+  return (temperature - 273.15).toFixed(0);
 }
 
 function mapModelToView(model) {
   return {
     description: _.get(model, 'weather[0].main'),
-    tempMin: convertKelvinToCelsius(_.get(model, 'main.temp_min')),
-    tempMax: convertKelvinToCelsius(_.get(model, 'main.temp_max')),
+    tempMin: convertKelvinToCelsius(_.get(model, 'main.temp_min')) + degreeSign,
```

Comment:
extract `tempMin`, `tempMax` & `temp` to consts above. There is too much calculation inside the return itself.